slack-base/src/main/java/com/hubspot/slack/client/models/events/bot/SlackAppHomeOpenedEventIF.java

```diff
@@ -0,0 +1,36 @@
+package com.hubspot.slack.client.models.events.bot;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+import com.hubspot.slack.client.methods.interceptor.HasUser;
+import com.hubspot.slack.client.models.events.SlackEvent;
+import com.hubspot.slack.client.models.response.views.HomeTabViewResponseIF;
+import org.immutables.value.Value;
+
+import java.util.Optional;
+
+@Value.Immutable
+@HubSpotStyle
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+@JsonDeserialize(as = SlackAppHomeOpenedEvent.class)
+public interface SlackAppHomeOpenedEventIF extends SlackEvent, HasUser {
+    String getTab();
+
+    Optional<HomeTabViewResponseIF> getView();
```

Comment:
You should reference the object type `HomeTabViewResponse` rather than the interface.